backend/ohq/management/commands/avg_queue_wait.py

```diff
@@ -0,0 +1,24 @@
+from django.db.models import Avg, F
```

Comment:
In general, I think it makes sense to split out the logic of a command into a different file (just so it's possible to call that function directly instead of being forced to use the command). Do you think it makes sense to move the logic for all of these into a `ohq/statistics.py` or something like that?

Reply:
Yep will do! After the changes, the commands that need to be run are queue_weekly_stat (groups weekly stats), queue_heatmap_stat (groups heatmap stats), and wait_time_days.